packages/thriver-core/lib/templates/browseHappy.html

```diff
@@ -0,0 +1,16 @@
+<template name="browseHappy">
+    <p class="browsehappy"><span class="inner"><span class="fa">&#xf071;</span> You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</span></p>
+    <!--[if IE 6]>
+                <p class="browsehappy"><span class="inner"><span class="fa">&#xf071;</span> You are using an outdated browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</span></p>
+    <![endif]-->
+    <!--[if IE 7]>
+                <p class="browsehappy"><span class="inner"><span class="fa">&#xf071;</span> You are using an outdated browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</span></p>
+    <![endif]-->
+    <!--[if IE 8]>
+                <p class="browsehappy"><span class="inner"><span class="fa">&#xf071;</span> You are using an outdated browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</span></p>
+    <![endif]-->
+    <!--[if IE 9]>
+                <p class="browsehappy"><span class="inner"><span class="fa">&#xf071;</span> You are using an outdated browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</span></p>
+    <![endif]-->
+    <!-- BROWSE HAPPY ENDS HERE -->
```

Comment:
The CSS conditions seem to be redundant...  Since the uncommented block at the top is fundamentally identical to each condition, they all seem unnecessary (not to mention element `<p class="browsehappy">` will display twice in these browsers with these conditions.)